File: insert.php
<?php
	include('connection.php');

    //$cn = pg_connect("host=localhost port=5432 dbname=student user=postgres password=admin");
    
    if(!$cn)
    {
        echo "Error:Unable to open database\n";
    }
    else
        echo "connected";

    $rollno= $_POST["rollno"];
    $name = $_POST["name"];
    $dob = $_POST["dob"];
    $age = $_POST["age"];
    $gender = $_POST["gender"];
    $phone = $_POST["phone"];
    $query = "INSERT INTO student VALUES('$rollno','$name', '$dob','$age','$gender','$phone')";
    $result= pg_query($cn,$query);
    header("Location: index.html");
    
?>

File: insert.py
import psycopg2


hostname = 'localhost'
database = 'student'
username = 'postgres'
pwd = 'admin'
post_id = 5432


try:
    conn = psycopg2.connect(host = hostname, dbname=database, user= username,password=pwd, port = post_id)
    cur = conn.cursor()
    cur.execute("INSERT INTO student values()")
    conn.commit()
    

except Exception as error:
    print(error)

finally:
    if cur is not None:
        cur.close()
    if conn is not None:
        conn.close()